crates/db_views/custom_emoji/src/api.rs

```diff
@@ -36,14 +34,14 @@ pub struct DeleteCustomEmoji {
 #[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Eq, Hash)]
 #[cfg_attr(feature = "ts-rs", derive(ts_rs::TS))]
 #[cfg_attr(feature = "ts-rs", ts(optional_fields, export))]
-/// Edit  a custom emoji.
+/// Edit a custom emoji.
 pub struct EditCustomEmoji {
   pub id: CustomEmojiId,
-  pub category: String,
-  #[cfg_attr(feature = "ts-rs", ts(type = "string"))]
-  pub image_url: Url,
-  pub alt_text: String,
-  pub keywords: Vec<String>,
+  pub category: Option<String>,
+  pub shortcode: Option<String>,
+  pub image_url: Option<DbUrl>,
+  pub alt_text: Option<String>,
+  pub keywords: Option<Vec<String>>,
```

Comment:
This needs to use `Option<Option<...>>` to avoid overwriting existing values, right?